vlib/v/checker/tests/anon_missing_generic_err.out

```diff
@@ -0,0 +1,21 @@
+vlib/v/checker/tests/anon_missing_generic_err.vv:8:2: notice: uninitialized `fn` struct fields are not allowed, since they can result in segfaults; use `?fn` or `@[required]` or initialize the field with `=` (if you absolutely want to have unsafe function pointers, use `= unsafe { nil }`)
+    6 | 
+    7 | struct Factory {
+    8 |     build fn () !
+      |     ~~~~~~~~~~~~~
+    9 | }
+   10 |
+vlib/v/checker/tests/anon_missing_generic_err.vv:21:14: error: original `factory` is a function that returns !E. You must add the generic type to generic type list.
+   19 | fn use_factory[F, E](factory fn (dep F) !E) Factory {
+   20 |     return Factory{
+   21 |         build: fn [factory] [F]() ! {
```

Comment:
It was not clear to me, what the generic type that should be added is.
Can you please modify the end of the error to be:
```
You must add the generic type `E` to generic type list, that is currently `[F]`.
```

Reply:
or
```
Add the generic type `E` to the generic type list that follows
```

Reply:
I wanted just the end of the error to be modified, the start was fine.